<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWindow</class>
 <widget class="QMainWindow" name="CalibrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>3D Viewer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="CloudViewer" name="myCloudViewer" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Adafruit">
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Adafruit</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetAdafruit_Contents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QPushButton" name="AdaFruit_Close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="Adafruit_ListPoses">
       <property name="text">
        <string>X Y Z Qw Qx Qy Qz</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Adafruit_Grab">
       <property name="text">
        <string>GrabData</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="Adafruit_Open">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Adafruit_Init">
       <property name="text">
        <string>Init</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="Adafruit_SavePoses">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="Adafruit_EmptyList">
       <property name="text">
        <string>EmptyList</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Adafruit_getCal">
       <property name="text">
        <string>getCal</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Kinect">
   <property name="windowTitle">
    <string>Kinect</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidget_Kinect_Contents"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_NexCave">
   <property name="windowTitle">
    <string>NexCave</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidget_NexCave_Contents">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="NexCave_ListPoses">
       <property name="text">
        <string>X Y Z Qw Qx Qy Qz</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="NexCave_Grab">
       <property name="text">
        <string>GrabData</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="NexCave_Close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="NexCave_Open">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="NexCave_SavePoses">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="NexCave_EmptyList">
       <property name="text">
        <string>EmptyList</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CloudViewer</class>
   <extends>QWidget</extends>
   <header>Widgets/cloudviewer.h</header>
   <container>1</container>
   <slots>
    <slot>ShowCoordinateSystem(double,Eigen::Affine3f,std::string)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>CalibrationWindow</sender>
   <signal>ShowCoordinateSystem(double,Eigen::Affine3f,std::string)</signal>
   <receiver>myCloudViewer</receiver>
   <slot>ShowCoordinateSystem(double,Eigen::Affine3f,std::string)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>707</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>753</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>ShowCoordinateSystem(double,Eigen::Affine3f,std::string)</signal>
 </slots>
</ui>
